### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by FrancescoBB** [InboxSDK/InboxSDK#1068]
**Frequency**: only for a limited number of users among the ones using the extension which makes use of Inbox SDK, but for those users the issue is always happening (no random behavior)

**Version**: tested with the last one available (see screenshots below)

**Issue Description**: when loading components related to a single email by using the methods

1- sdk.Toolbars.registerToolbarButtonForList
2- sdk.Conversations.registerMessageViewHandler

the button and the sidebar are not loaded at all and the two errors in the screenshots below appears (sometimes the first one, sometimes the second one, depending on which mail is selected, with no apparent pattern dictating which one happens when)

![image](https://github.com/InboxSDK/InboxSDK/assets/62301150/beaf312b-2046-4ce5-9a8e-3c9479a36cb6)
![image](https://github.com/InboxSDK/InboxSDK/assets/62301150/10d06466-352d-4bc8-8e03-8be001501c1d)

**Expected behavior**:

Buttons and sidebar appearing correctly as it happens with most of the users

Other Notes:

1- Issue started on 2023-11-14
2- All the users of the workgroup belongs to the same gmail domain 
3- The issue happens both in normal mode and in delegated mode (while the not affected users are ok both in normal and delegated mode)
4- No relavant differences in the gmail settings among the users

**Comment by aleemstreak** [comment]
Do you have access to an account that is exhibiting this behaviour?
ᐧ

**Comment by FrancescoBB** [comment]
@aleemstreak yes, it's an account used by one of our customers and I've been able to reproduce the issue yesterday by using the user's account.

**Comment by aleemstreak** [comment]
is that something you could share with us?

**Comment by FrancescoBB** [comment]
@aleemstreak no, unfortunately I cannot share the account as it is the customer's private one.
But I can access it and take some logs if it can be of help for you.
In case, which ones would you need?

**Comment by wegry** [comment]
@FrancescoBB #1073 may help with the issue.

**Comment by FrancescoBB** [comment]
Thanks @wegry we'll give it a try asap!

**Comment by FrancescoBB** [comment]
@wegry the fix you referenced solved the problem for our user!
Many thanks for the fast intervention!
